packages/flutter_tools/lib/src/runner/flutter_command.dart

```diff
@@ -106,6 +106,7 @@ class FlutterOptions {
   static const String kEnableExperiment = 'enable-experiment';
   static const String kFileSystemRoot = 'filesystem-root';
   static const String kFileSystemScheme = 'filesystem-scheme';
+  static const String kDebuggingInfoOption = 'save-debugging-info';
```

Comment:
This flag not only causes the debugging info to be saved off to the side, but also causes the VM to print debugging info in a different format, so I think we should consider a more descriptive flag name. Maybe `--compress-debug-info`, or `--split-debug-info`, or `--separate-debug-info`.

Reply:
I like split because it is easier to spell than ~separerate~ separate